MapboxMobileEvents/MMEAPIClient.m

```diff
@@ -116,6 +116,9 @@ - (void)setBaseURL:(NSURL *)baseURL {
 #pragma mark - Utilities
 
 - (NSError *)statusErrorFromRequest:(NSURLRequest *)request andHTTPResponse:(NSHTTPURLResponse *)httpResponse {
```

Comment:
Probably a stupid question, but can `(NSURLRequest *)request `param be `nil`? And what will happen if it is?

Reply:
Hasn't happened yet AFAIK but adding in some precautions won't hurt. 👍